karma.config.cjs

```diff
@@ -9,7 +11,7 @@ module.exports = function(config) {
     port: 9876,
     colors: true,
     logLevel: config.LOG_INFO,
-    browsers: ['ChromeHeadless', 'FirefoxHeadless'],
```

Comment:
This makes me wonder if there's a similar dep to `chromium` but for firefox? Would be great to continue testing for Firefox.

Reply:
I can look into it! Alternatively is there maybe a `CODESPACES` (or simmer) env var?

Reply:
I did a bit of digging wasn't able to find an easy Firefox or gecko npm package. I'll keep my eyes peeled though and maybe something will turn op and we can add it later. Agreed it would be nice to run in more then chromium!